src/app/shared/table/actions/table-view-pricing-definitions-action.ts

```diff
@@ -8,20 +8,20 @@ import { PricingButtonAction, PricingDefinitionDialogData } from '../../../types
 import { TableActionDef } from '../../../types/Table';
 import { TableViewPricingListAction } from './table-pricing-action';
 
-export interface TableViewPricingsActionDef extends TableActionDef {
+export interface TableViewPricingDefinitionsActionDef extends TableActionDef {
   action: (pricingDialogComponent: ComponentType<unknown>, dialog: MatDialog,
     dialogParams: DialogParams<PricingDefinitionDialogData>, refresh?: () => Observable<void>) => void;
 }
-export class TableViewPricingsAction extends TableViewPricingListAction {
-  public getActionDef(): TableViewPricingsActionDef {
+export class TableViewPricingDefinitionsAction extends TableViewPricingListAction {
```

Comment:
Should extend TableViewAction?
Missing Edit action that extends TableEditAction?